src/code/client.ts

```diff
@@ -545,8 +550,9 @@ class CloudFileManagerClient {
             if (err) {
               return this.alert(err, () => this.ready())
             }
-            // if we just opened the file, it doesn't need to be saved until the contents are changed
-            const additionalState = { openedContent: content.clone(), saved: true }
+            // if we just opened the file, it doesn't need to be saved until the contents are changed unless
+            // it requires conversion from an older version
+            const additionalState = { openedContent: content.clone(), dirty: content.requiresConversion() }
```

Comment:
>If you are around in the next hour or so, I would love to just quickly get your feedback on small CFM change that Doug and I just made.

>The `saved: true` seemed to be triggering a UI update saying that *all changes have been saved* which was confusing us in QA because we were thinking that a save should not have been triggered.
>We also set the dirty bit check if the loaded content needs to be updated. A similar thing is done elsewhere in the client when the content updates.

Seems right to me. You never know if there might be other downstream code that is relying on this behavior somehow, but I bet whoever added that `saved: true` was thinking of it as a dirty flag. In any case, this seems like the correct behavior of this function and we can deal with any downstream consequences that might arise.